packages/components/src/components/mobile-drawer/mobile-drawer.scss

```diff
@@ -148,6 +148,9 @@
             top: 0;
             left: 0;
             overflow: auto;
+            &:has(.settings-language__language-container) {
+                z-index: 7;
```

Comment:
just curios.. will it work if we simply give a z-index without `&:has(.settings-language__language-container)` this part?